civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1567,6 +1567,27 @@ function civihr_employee_portal_menu() {
         'access arguments' => array('manage hrreports settings'),
         'type' => MENU_CALLBACK,
     );
+    $items['reports/%/configuration/%'] = array(
+        'title' => 'Get Report configuration',
+        'page callback' => 'civihr_employee_portal_get_report_configuration_json',
+        'page arguments' => array(1, 3),
+        'access arguments' => array('access hrreports'),
+        'type' => MENU_CALLBACK,
+    );
+    $items['reports/%/configuration/%/save'] = array(
+        'title' => 'Save Report configuration',
+        'page callback' => 'civihr_employee_portal_save_report_configuration_json',
```

Comment:
I am not really a drupal expert , but when I worked with employee portal I was thinking that maybe there is a better way to organize stuff instead of creating all the methods in  extension.module file , so these callbacks can be placed in a separate file or something .. don't know actually I will let it for you to decide